test/integration/discord.test.js

```diff
@@ -75,4 +75,70 @@ describe("test discord actions", function () {
         });
     });
   });
+
+  describe("test discord actions for nickname for verified user", function () {
+    beforeEach(async function () {
+      fetchStub = sinon.stub(global, "fetch");
+      const user = { ...userData[4], discordId: "123456789" };
```

Comment:
any reason for specifically choosing `userData[4]`?
If the reason is SUPER_USER, then we may name the variable appropriately :)

Reply:
no I just chose it and added discord id. I can change it if you suggest.

Reply:
I have updated